src/courses/models.py

```diff
@@ -205,6 +205,11 @@ class Semester(models.Model):
     QUEST_AWAITING_APPROVAL = -2
     STUDENTS_WITH_NEGATIVE_XP = -3
 
+    name = models.CharField(
+        blank=True, unique=False, max_length=50,
+        help_text="If empty, will instead use First day as it's name"
```

Comment:
"If left blank, the semester will display it's First Day as a default name, in the form: mmm-YYYY."